src/utilstrencodings.h

```diff
@@ -22,7 +22,15 @@
 /** This is needed because the foreach macro can't get over the comma in pair<t1, t2> */
 #define PAIRTYPE(t1, t2)    std::pair<t1, t2>
 
-std::string SanitizeString(const std::string& str);
+static const int SAFE_CHARS_DEFAULT = 0; //!< Default rule in SanitizeString()
```

Comment:
+1 for using enum. This is a perfect fit for one.

Not sure passing in the characters in a string is better. The idea makes sense, but having it like this keeps open the possibility of optimizing SanitizeString to use something else than string.find() on every character.